docs/messaging.md

```diff
@@ -68,6 +68,15 @@ mosquitto_sub -u <thing_id> -P <thing_key> -t channels/<channel_id>/messages -h
 If you are using TLS to secure MQTT connection, add `--cafile docker/ssl/certs/ca.crt`
 to every command.
 
+###### Subtopic's
+In order to use MQTT subtopic's and extend the channel with subtopic or name the channel using your own naming convention, You can simply add any prefix on base `/messages` topic.
+
+Example subtopic publish/subscribe for bedroom temperature would be
+
+ `channels/<channel_id>/messages/bedroom/temperature`
+
+ Subtopic's are generic and multilevel, You can use any prefix with any level depth. However for the sake of simplicity we recommend to keep it simple and design your topic's at max 2 or 3 level depth.
+
```

Comment:
I didn't added nothing NATS related because we don't mention NATS at all (the possibility to consume NATS topic directly), maybe this is not a perfect place for that. But Yes its good idea, I will add it here for now.

Reply:
I think we can add it here for now, and then we will move the whole chapter.